Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -271,9 +271,18 @@ public void GetAllRevitParamsAndIds(Base speckleElement, DB.Element revitElement
       speckleElement["builtInCategory"] = builtInCategory.ToString();
 
       //NOTE: adds the quantities of all materials to an element
-      var qs = MaterialQuantitiesToSpeckle(revitElement, speckleElement["units"] as string);
-      if (qs != null)
-        speckleElement["materialQuantities"] = qs;
+      List<MaterialQuantity>? materialQuantities = null;
+      try
+      {
+        materialQuantities = MaterialQuantitiesToSpeckle(revitElement, speckleElement["units"] as string)?
+          .ToList();
+      }
+      catch (Exception e)
+      {
+        SpeckleLog.Logger.Error(e, "Error retrieving material quantities from element of type {elementType} and category {elementCategory}", revitElement.GetType(), revitElement.Category);
+      }
+      if (materialQuantities != null)
+        speckleElement["materialQuantities"] = materialQuantities;
     }
```

Comment:
We can simplify this by moving the final `if` statement into the try/catch.
Would allow us to remove the null list initialisation at the top and just do `var materialQuantities = ...` in line 277.